packages/hydrojudge/src/hosts/hydro.ts

```diff
@@ -189,6 +211,7 @@ export default class Hydro implements Session {
                     compilerVersionCallback();
                 });
             }
+            if (request.sync) this.syncData(request.sync.domainId, request.sync.docId, request.sync.files, request.sync.taskId);
```

Comment:
should also be added to a queue to prevent too many parallel downloads

Reply:
I reused the function `_cacheOpen`, it already has a queue, so i don't need to implement it again, i guess?